src/pages/QuickMenu/QuickMenu.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+// import CsCenter from './CsCenter/CsCenter';
+import { Link } from 'react-router-dom';
+import Common from '../../components/Common/Common';
+import styled from 'styled-components';
+
+const QuickMenu = () => {
+  const { ImgGobal, IcoMovie } = Common;
+
+  return (
+    <>
+      <TicketWrap />
+
+      <TotalWrap>
+        <InnerTotal>
+          <IcoMovie width="105px" height="105px" backgroundPosition="0 -350px">
+            영화선택
+          </IcoMovie>
+          <MovieWrap>
+            <ImgWrap>
+              <ImgGobal src="../../images/pooh.png" alt="영화이미지" />
+            </ImgWrap>
+            <ImgDetail>
+              <DetailMain>웅남이</DetailMain>
+              <DetailDes>SCREENX 2D</DetailDes>
+              <DetailAge>19세 관람가</DetailAge>
+            </ImgDetail>
+          </MovieWrap>
+          <InfoTicket>
+            <TitTicket>극장 : </TitTicket>
+            <TxtTicket>CGV 구로</TxtTicket>
+            <TitTicket>일시 : </TitTicket>
+            <TxtTicket>2023.3.31(금)11:00</TxtTicket>
+            <TitTicket>상영관 : </TitTicket>
+            <TxtTicket>1관 SCREENX 6층</TxtTicket>
+            <TitTicket>인원 : </TitTicket>
+            <TxtTicket>일반 1명</TxtTicket>
+          </InfoTicket>
+          <InfoTicket>
+            <TitTicket>좌석명 : </TitTicket>
+            <TxtTicket>일반석</TxtTicket>
+            <TitTicket>좌석번호 : </TitTicket>
+            <TxtTicket>E8</TxtTicket>
+          </InfoTicket>
+          <InfoTicket>
+            <TitTicket>일반 : </TitTicket>
+            <TxtTicket>12,000원 X 1</TxtTicket>
+            <TitTicket>총금액 : </TitTicket>
+            <TxtTicket>12,000원</TxtTicket>
+          </InfoTicket>
+
+          <IcoMovie
+            width="105px"
+            height="105px"
+            backgroundPosition="-120px -240px"
+          >
+            영화선택
+          </IcoMovie>
+        </InnerTotal>
+      </TotalWrap>
+    </>
+  );
+};
+
+const TicketWrap = styled.div`
+  background-color: #f2f0e4;
+  width: 996px;
+  height: 600px;
+  margin: 0 auto;
+`;
+const TotalWrap = styled.div`
+  background-color: #222;
+  height: 128px;
+`;
+const InnerTotal = styled.div`
+  width: 996px;
+  margin: 0 auto;
+  padding: 10px 0;
+  display: flex;
+  text-align: left;
+  justify-content: space-between;
+  color: #ccc;
+`;
+const MovieWrap = styled.div`
+  display: flex;
+  padding: 10px;
+`;
+const ImgWrap = styled.div`
+  width: 70px;
+`;
+const ImgDetail = styled.div`
+  font-size: 12px;
+  padding: 0 0 0 10px;
+`;
+const DetailMain = styled.div``;
+const DetailDes = styled.div`
+  padding: 10px 0 10px 0;
+`;
+const DetailAge = styled.div``;
+const InfoTicket = styled.dl`
```

Comment:
```suggestion
const ImgDetail = styled.div`
  font-size: 12px;
  padding: 0 0 0 10px;
`;

const DetailMain = styled.div``;

const DetailDes = styled.div`
  padding: 10px 0 10px 0;
`;

const DetailAge = styled.div``;

const InfoTicket = styled.dl`
```

스타일끼리 다 붙어있어서 가독성이 너무 떨어집니다!
해당 부분 전부 떼어주세요!

Reply:
네! 알겠습니다!